pkg/gofr/gofr.go

```diff
@@ -310,86 +309,97 @@ func (a *App) initTracer() {
 	traceExporter := a.Config.Get("TRACE_EXPORTER")
 	tracerURL := a.Config.Get("TRACER_URL")
 
-	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions
+	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions.
 	tracerHost := a.Config.Get("TRACER_HOST")
 	tracerPort := a.Config.GetOrDefault("TRACER_PORT", "9411")
+	authHeader := a.Config.Get("TRACER_AUTH_KEY")
```

Comment:
TRACER_AUTH_KEY is not deprecated, it should be clearer, by inverting and separating

```suggestion
	authHeader := a.Config.Get("TRACER_AUTH_KEY")

	// deprecated : tracer_host and tracer_port are deprecated and will be removed in upcoming versions.
 	tracerHost := a.Config.Get("TRACER_HOST")
 	tracerPort := a.Config.GetOrDefault("TRACER_PORT", "9411")
```